neurom/features/neuronfunc.py

```diff
@@ -211,12 +211,11 @@ def _sort_angle(p1, p2):
             for i, _ in enumerate(ordered_vectors)]
 
 
-@feature(shape=(...,))
 def sholl_crossings(neurites, center, radii):
     """Calculate crossings of neurites.
 
     Args:
-        nrn(morph): morphology on which to perform Sholl analysis
+        nrn(morph): morphology on which to perform Sholl analysis, or list of sections
```

Comment:
update docstring according to function arguments

Reply:
that seem to be missing from before, I added it, good catch!